packages/cli/src/tools/config/resolveNodeModule.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @flow
+ */
+import path from 'path';
+
+/**
+ * Finds a path inside `node_modules`
+ */
+export default function resolveNodeModule(
```

Comment:
Hm, not sure about the name, maybe `resolveNodeModueDir` would be closer to what it does? We're not resolving a module (a JS file) but the directory it's in. What do you think?

Reply:
Makes sense, updated